tests/backends/test_vanilla_django_backend.py

```diff
@@ -110,6 +122,16 @@ def test_render_inheritance_email(self):
         self.assertEqual(PLAIN_RESULT, response['plain'])
         self.assertEqual('Another subject for vintasoftware', response['subject'])
 
+    def test_email_text_escaping(self):
+        self.context['username'] = '<p>vintasoftware</p>'
+        self.context['subject'] = 'Subject with tag <p>'
```

Comment:
I think this is not being used/asserted

Reply:
It is used as argumento to the method _render_email right below and it changes the response content

Reply:
actually the subject really wasn't being used